### PR TITLE
Loading Screen

### PR DESCRIPTION
**Pull request by dborg291**
**Purpose:**
Create a loading component to shown during loading events.

**Changes:**
1. Created loading component, `Loading.jsx`
2. Display the loading screen during authentication when the request is being made.

**Testing:**
Tested with successful and failing authentication on both Android and iOS.

**Screenshots:**

<details>
<summary> Andriod </summary>
image
</details>
<details>
<summary> iOS </summary>
None
</details>

### CODE REVIEW

**Review by robinsph21 — COMMENTED** [review]
Reviewed, but I think we'll discuss more in work meeting, as I don't know if we will need linear gradient at all. We're also waiting on comments from Ben regarding themes and the style guide.

File: screens/Loading.jsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { ActivityIndicator, StyleSheet, View, Image } from 'react-native';
+import { LinearGradient } from 'expo-linear-gradient';
+import Logo from '../assets/logo.png';
+
+export default function loadingScreen() {
+  return (
+    <View>
+      <LinearGradient
+        colors={['#21BDC1', '#41D06A']}
```
Comment (brmscheiner) [inline screens/Loading.jsx:10]: Colors should eventually be coming from a constants file

File: screens/Loading.jsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        style={styles.content}
+        start={[0, 0]}
+        end={[1, 0]}
+      >
+        <Image source={Logo} />
+        <ActivityIndicator
+          style={styles.spinner}
+          size="large"
+          color="#2C2C2C"
```
Comment (brmscheiner) [inline screens/Loading.jsx:19]: as above

**Review by brmscheiner — APPROVED** [review]

**Comment by brmscheiner** [comment]
Agreed with @robinsph21 re: theme/style changes coming eventually. Still nice to get the logic in and adjust the details later

**Comment by dborg291** [comment]
> Reviewed, but I think we'll discuss more in work meeting, as I don't know if we will need linear gradient at all. We're also waiting on comments from Ben regarding themes and the style guide.

Goal of PR was mainly to add the functionality of the loading screen. Planned o changing UI when plan were finalized. I'll hold off on merging until them.

**Pull request merged**